pkg/webui/console/containers/side-bar/index.js

```diff
@@ -31,15 +31,28 @@ import SwitcherContainer from './switcher'
 
 import style from './side-bar.styl'
 
-const getViewportWidth = () =>
-  Math.max(document.documentElement.clientWidth || 0, window.innerWidth || 0)
-
-const Sidebar = ({ isDrawerOpen }) => {
-  const viewportWidth = getViewportWidth()
-  const isMobile = viewportWidth <= LAYOUT.BREAKPOINTS.M
```

Comment:
I think we were talking about this before but the issue with a solution is that this only calculates once during render. However, this would not update if the screen size changes in between renders. For example, if you render in mobile screen size and then resize the browser window to desktop, the sidebar will still think it is `isMobile` until it renders again.

Reply:
Generally speaking, using such props like `isMobile` is typically unnecessary because most of the things that such a prop is used for can be solved by using media-queries instead. In this particular case, I've decided to leave it though since it enables to render tooltip dropdowns only when necessary (instead of hiding them via media-queries but still running mouse-in/out hooks).